pkg/utils/status.go

```diff
@@ -162,7 +162,7 @@ func getDaemonSetStatus(ctx context.Context, instance *csmv1.ContainerStorageMod
 
 		failedCount := 0
 		podList := &corev1.PodList{}
-		label := instance.GetNamespace() + "-node"
+		label := instance.GetName() + "-node"
```

Comment:
Have other drivers tested this?
These changes will impact all drivers, right?

Reply:
powerflex and powerscale -- yes, all drivers use this code

Reply:
Can we test/have other drivers test to make sure? 
Moose team has had a lot of operator defects lately- would like to not assume things will work without testing 
Once we fix e2e testing we won't need to explicitly consult other teams for each change

Reply:
tested on other drivers as well, merging with changes that came in from PR this morning and then should be ready to go